### PR TITLE
Add debugging aids

### PR DESCRIPTION
**Pull request by samgdotson**
## Please review and merge PR #39 before reviewing this one!

# Update
This PR primarily introduces debugging aids in the `CapacityExpansion` and `Technology` classes. I also added the ability to turn a list of technology objects into a pandas dataframe.

### CODE REVIEW

**Comment by samgdotson** [comment]
@nsryan2 @yardasol @LukeSeifert @abachma2 would one of you mind taking a look at this?

File: osier/__init__.py
Comment (yardasol) [inline osier/__init__.py:15]: What's this for?

File: osier/equations.py
Comment (yardasol) [inline osier/equations.py]:
```suggestion
    or the electricity demand for a dispatch model with weighted 
```

File: osier/equations.py
Comment (yardasol) [inline osier/equations.py]:
```suggestion
    attribute : {'demand', 'volatility'}
        Indicates whether to calculate the volatility of electricity demand
        or electricity price.
```
Don't need to specify the default in the docstring since it's in the function signature.

File: osier/models/capacity_expansion.py
Comment (yardasol) [inline osier/models/capacity_expansion.py:94]: add this to the docstring.

File: osier/models/capacity_expansion.py
Comment (yardasol) [inline osier/models/capacity_expansion.py]: This is not super readable. It looks like a typo. Maybe something like `print_problem_formulation`?

File: osier/technology.py
Comment (yardasol) [inline osier/technology.py]:
```suggestion
        Specifies the rate at which carbon dioxide is emitted during operation.
```
Also, what units are these in? Is there a parameter to modify it? I assume it's the same for the units for `co2_rate`...

**Review by yardasol — CHANGES_REQUESTED** [review]
Hi @samgdotson,
Nice clean PR. I have a few comments, but otherwise looks good!

File: osier/__init__.py
Comment (samgdotson) [reply osier/__init__.py:15]: This unit doesn't exist by default in `unyt`, so I make sure that it's defined locally because it's used elsewhere in `osier`.

File: osier/technology.py
Comment (samgdotson) [reply osier/technology.py]: Yes, it is modifiable using lifecycle_co2_rate

**Comment by samgdotson** [comment]
Thanks for the review @yardasol! I think I have addressed your comments.

**Review by yardasol — APPROVED** [review]
Good work @samgdotson. I'm gonna wait for some other folks to do a review before merging.

**Comment by samgdotson** [comment]
@yardasol I think you can go ahead and merge this since others are reviewing #41

**Comment by nsryan2** [comment]
It all looks good to me!

**Review by nsryan2 — APPROVED** [review]
LGTM!

**Pull request merged**
